Czech/RenX_Game.cze

```diff
@@ -0,0 +1,1598 @@
+
+[Rx_Weapon_Airstrike_GDI]
+ItemName="Nálet A-10"
+PickupMessage="Nálet A-10"
+
+[Rx_Weapon_Airstrike_Nod]
+ItemName="Nálet AC-130"
+PickupMessage="Nálet AC-130"
+
+[Rx_Weapon_TiberiumFlechetteRifle]
+ItemName="Tibériový Šipkostříl"
+PickupMessage="Tibériový Šipkostříl"
+
+[Rx_Weapon_TiberiumAutoRifle]
+ItemName="Tibériová Auto-Puška"
+PickupMessage="Tibériová Auto-Puška"
+
+[Rx_Weapon_TiberiumAutoRifle_Blue]
+ItemName="Tibériová Auto-Puška"
+PickupMessage="Tibériová Auto-Puška"
+
+[Rx_Weapon_ATMine]
+ItemName="Protitanková Mina"
+PickupMessage="Protitanková Mina"
+
+[Rx_Weapon_EMPGrenade]
+ItemName="EMP Granát"
+PickupMessage="EMP Granát"
+
+[Rx_Weapon_EMPGrenade_Rechargeable]
+ItemName="EMP Granát"
+PickupMessage="EMP Granát"
+
+[Rx_Weapon_RepairTool]
+ItemName="Opravovací Zbraň"
+PickupMessage="Opravovací Zbraň"
+PurchaseDesc="<font size='8'>Plusy:\n-Opraví jednotky/budovy\n-Zneškodní miny\n\nMinusy:\n-Musí se znovu nabít\n-POUŽIJE SLOT VYBAVENÍ  </font>"
+			
+[Rx_Weapon_RepairTool_Crate]	
+ItemName="Pokročilá Opravovací Zbraň"	
+PickupMessage="Pok. Opravovací Zbraň"
+
+[Rx_Weapon_SmokeGrenade]
+ItemName="Kouřový Granát"
+PickupMessage="Kouřový Granát"
+
+[Rx_Weapon_SmokeGrenade_Rechargeable]
+ItemName="Kouřový Granát"
+PickupMessage="Kouřový Granát"
+
+[Rx_Weapon_TiberiumGrenade_Rechargeable]
+ItemName="Tibériový Granát"
+PickupMessage="Tibériový Granát"
+
+[Rx_Weapon_SniperRifle_UnSilenced]
+ItemName="Odstřelovací Puška"
+PickupMessage="Odstřelovací Puška"
+
+//=====================================
+//Rx WeaponAbilities
+//=====================================
+
+[Rx_WeaponAbility]
+ItemName="Bezejmenná Schopnost"
+PickupMessage="Bezejmenná Schopnost"
+
+[Rx_WeaponAbility_TiberiumGrenade]
+ItemName="Tibériový Granát"
+PickupMessage="Tibériové Granáty"
+
+[Rx_WeaponAbility_Grenade]
+ItemName="Tříštivý Granát"
+PickupMessage="Tříštivé Granáty"
+
+[Rx_WeaponAbility_EMPGrenade]
+ItemName="EMP Granát"
+PickupMessage="EMP Granáty"
+
+[Rx_WeaponAbility_SmokeGrenade]
+ItemName="Kouřový Granát"
+PickupMessage="Kouřové Granáty"
+
+[Rx_WeaponAbility_TacRifleGrenade]
+ItemName="Taktická Puška [Granát]"
+PickupMessage="Odpalovač Granátů"
+
+//=====================================
+//	Damage Types
+//=====================================
+
+
+[Rx_DmgType]
+DeathString="`k zabil `o"
+FemaleSuicide="`o spáchala sebevraždu"
+MaleSuicide="`o spáchal sebevraždu"
+
+[Rx_DmgType_A10_Missile]
+DeathString="`o byl zasažen střelou z náletu hráče `k"
+MaleSuicide="`o nedokázal utéct z oblasti náletu"
+WeaponName="A10 Airstrike"
+
+[Rx_DmgType_AATower]
+DeathString="`o se dostal do vzdušného prostoru hráče `k"
+
+[Rx_DmgType_Abduction]
+DeathString="`o se ztratil ve vesmíru"
+MaleSuicide="`o se ztratil ve vesmíru"
+WeaponName="Abducted"
+
+[Rx_DmgType_AGT_MG]
+DeathString="`o was caught inside `k perimeter and was terminated immediately"
```

Comment:
missing translation

Reply:
I haven't finished translating the damage types. Are they even in use yet?